app/assets/javascripts/discourse/app/components/post-text-selection-toolbar.gjs

```diff
@@ -33,25 +33,35 @@ export default class PostTextSelectionToolbar extends Component {
       {{this.appEventsListeners}}
     >
       <div class="buttons">
-        {{#if this.embedQuoteButton}}
-          <DButton
-            @icon="quote-left"
-            @label="post.quote_reply"
-            @title="post.quote_reply_shortcut"
-            class="btn-flat insert-quote"
-            @action={{@data.insertQuote}}
-          />
+        {{#if this.showPrimaryActions}}
+          {{#if this.embedQuoteButton}}
+            <DButton
+              @icon="quote-left"
+              @label="post.quote_reply"
+              @title="post.quote_reply_shortcut"
+              class="btn-flat insert-quote"
+              @action={{@data.insertQuote}}
+            />
+          {{/if}}
+
+          {{#if @data.canEditPost}}
+            <DButton
+              @icon="pencil-alt"
+              @label="post.quote_edit"
+              @title="post.quote_edit_shortcut"
+              class="btn-flat quote-edit-label"
+              {{on "click" this.toggleFastEdit}}
+            />
+          {{/if}}
         {{/if}}
 
-        {{#if @data.canEditPost}}
-          <DButton
-            @icon="pencil-alt"
-            @label="post.quote_edit"
-            @title="post.quote_edit_shortcut"
-            class="btn-flat quote-edit-label"
-            {{on "click" this.toggleFastEdit}}
-          />
-        {{/if}}
+        <PluginOutlet
+          @name="post-text-selection-buttons-after"
```

Comment:
Not sure of the name here if its good, and I am open to suggestions on changing this.

Reply:
`post-text-after-primary-actions` would be good I think